src/common/common.js

```diff
@@ -1372,6 +1372,110 @@ const RandomTips = (function () {// eslint-disable-line no-unused-vars
     return me;
 })();
 
+const Colors = (() => { // eslint-disable-line no-unused-vars
+    const me = {};
+
+    me.ContrastBreakpoints = {
+        A: 2,
```

Comment:
No my numbers were not all the A/AA/AAA things…

But I'll change that by myself.